src/__tests__/Activity.test.ts

```diff
@@ -24,9 +24,9 @@ import { afterAll, beforeAll, expect, test, vi } from 'vitest'
 import { nextTick } from 'vue'
 
 import Activity from '../components/Activity.vue'
-import ActivityModel from '../models/ActivityModel.ts'
```

Comment:
IIRC, we agreed to keep file extensions in imports, so please add it back :)

Reply:
Yes, fixed it.